### ISSUE TITLE
[Discussion] Deprecate table

### ISSUE THREAD

**Issue by mpourismaiel** [okkur/syna#399]
Table fragment and table support in item fragment can be removed in favor of markdown table. One thing that would be missed is hidden columns in smaller devices and another thing would be alignment. What do you think?

**Comment by stp-ip** [comment]
I agree more and more. Our tables are not that readable. Markdown only ones are more limited, but styled nicely I think it's easier for the user.
Definitely needs more styling than comes by default.
For a functioning site something like this should be possible https://about.ejderrapgott.de/songs/.
So that would mean having the markdown as content in an item fragment. Styling should be similar I think.

**Comment by stp-ip** [comment]
I think this should be the first thing we tackle in v0.16.0.
This means with the v0.15.0 release, we should add a big note to demo code and the docs that the table fragment is deprecated and one should use markdown tables instead with new features for markdown tables coming in v0.16.0. What do you think?

Plain markdown support should be fine as a first step and then look into how a few "features" from the previous version could be done via configuration or inline snippets.

**Comment by stp-ip** [comment]
Currently markdown tables are not styled at all and just look like a huge blob of text. When moving to this, we should use our old design to support this. Just wanted to add this as context here.

**Comment by orchid00** [comment]
how can I add style to markdown tables? @stp-ip I need badly
